Tests/IBLinterKitTest/Utils/Config+LintablePathsTests.swift

```diff
@@ -2,7 +2,7 @@
 import XCTest
```

Comment:
This filename should be changed.

`Config+LintablePathsTests.swift` to `LintablePathsTests.swift` or something...